render_map/src/main.rs

```diff
@@ -296,46 +252,114 @@ fn process<E>(path: &Path, out_path: &Path, mut external: &mut E, config: &Confi
         }
     }
 
-    if let Some(ref c) = config.crop {
-        min_x = c.min_x;
-        min_y = c.min_y;
-        max_x = c.max_x;
-        max_y = c.max_y;
+    Ok(layers)
+}
+
+fn prepare_tilesets<E>(
+    layers: &Vec<Layer>,
```

Comment:
Here and in the other places a `&Vec<T>` is passed: It's generally good practice to use `&[T]` as parameter type instead of `&Vec<T>`, it allows strictly more possible parameters.